Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM linuxserver/docker-compose
+
+User root
+
+COPY . /workspace
+WORKDIR /workspace
+
+RUN apt-get update && \
+    apt-get install -y \
+    software-properties-common
+    
+RUN add-apt-repository ppa:deadsnakes/ppa
+
+RUN apt-get install -y \
+    python3.7 \
+    python3-pip
+
+RUN pip3 install -U -r requirements.txt
+
+RUN curl -O https://download.docker.com/linux/static/stable/x86_64/docker-18.03.1-ce.tgz && \
+    tar zxvf docker-18.03.1-ce.tgz && \
+    cp docker/docker /usr/local/bin/ && \
+    rm -rf docker docker-18.03.1-ce.tgz
```

Comment:
where's the mount?

Reply:
What mount do you want?

Reply:
I saw the ticket.
FYI
![image](https://user-images.githubusercontent.com/46285572/144448846-ac595cee-3017-4b06-9375-8102bdb66c11.png)

Reply:
Good I'm stupid thanks xDD